routes/keyword.js

```diff
@@ -7,19 +7,41 @@ const Keyword = require('../models/KeywordModel.js')
 
 const router = express.Router()
 
-router.get('/:word', (req, res) => {
-    console.log(req.params)
+router.get('/word/:word', (req, res) => {
     let word = req.params.word
-    console.log(word)
     Keyword.find({word: word}).lean().then((wordList) => {
-        console.log(wordList[0])
-        console.log(wordList[0].votes)
-        console.log(word)
         res.send(wordList[0].votes)
     }).catch((err) => {
-        console.log(err)
         res.status(500).end()
     })
 })
 
+router.get('/top', (req, res) => {
+	Keyword
+		.find()
+		.lean()
+		.then((wordList) => {
+			let topWords = wordList.reduce((accumulator, word) => {

```

Comment:
Why the `let`? If the variable isn't being changed you should make it a `const` to indicate the variable not being reminded to anything